app/services/moderator/banisher.rb

```diff
@@ -40,6 +44,8 @@ def add_banned_role
     end
 
     def delete_reactions
+      return unless user.reactions.count.positive?
```

Comment:
It's possible to reduce number of queries to 1 if the first check will be like this:
`user.reactions.load.size.positive?`
This will load the assosiation, and the second query won't be needed.

Another way to optimize is replacing `user.reactions.count.positive?` with `user.reactions.any?`
This way 2 queries will be executed, but the first will be `select 1` instead of `select count(*)`